internal/standby/standby.go

```diff
@@ -75,13 +85,72 @@ func (s *Service) RunMQTT(ctx context.Context) error {
 			}
 		})
 
+	s.subscribeToTopic(s.cfg.MQTT.ReadCommandTopic,
+		func(client mqtt.Client, msg mqtt.Message) {
+			s.logger.Debug(fmt.Sprintf("Received message: %s from topic: %s", msg.Payload(), msg.Topic()))
+			s.LatestCommandReceived = time.Now()
```

Comment:
I believe this is not thread-safe. Can we maintain a separate storage (perhaps in-memory, I don't know what limitations you have in this project) which operates predictably under concurrent access?

Reply:
ChatGPT suggested using `sync.Mutex` to ensure that only one goroutine can access the variable at a time. I implemented this with [get/set methods](https://github.com/EvergenEnergy/remote-standby/pull/9/files#diff-f270689238c6ffd464b5f3ed580853433bf7d955d5709d627440be7df625f690R180). Does that look like an acceptable solution?

Reply:
It does the trick, however I would still suggest to delegate the responsibility of maintaining and updating this state to some dependency (let's call it `storage`), rather than use a private field in the `Service`. Some reasons:
* Clearer separation of concerns
* Protection against accidental thread-unsafe access (compiler can't force you to only use those getter and setter methods).
* Ease of testing again.

Reply:
ok thanks, I've moved this into a separate `storageService`